### DISCUSSION TITLE
Clicking "retry" on a failed replay should bring player back to the map

### DISCUSSION THREAD

**Discussion by drkamaon** [ppy/osu#21870] [Ideas]
https://user-images.githubusercontent.com/61342323/209767994-550a44c7-9d47-4e30-8884-42456c433259.mp4

Here's a clip of me playing a random map, fail it, save the replay and hit retry.
That brought me back to the start of the replay, while the “watch replay” button do the same thing.
This isn't really a bug, but it somewhat gets in the way; the player has to press Esc and Enter to retry.

I think there should be a literal button that says “replay” on top of "retry", and do the same thing when watching a replay.

also please make f1 work

**Comment by JacksonChen666** [comment]
if you watch a failed replay (not after failed gameplat), you'll be presented with the fail screen.

the current behavior of retry is to watch the replay again. that can be useful in case the replay does something weird (like skimming during mania hold notes)

if the behavior is changed, someone who's just watching replays might unexpectedly play a map when they didn't mean to. so yeah uh, conditional retry behavior? i'm not sure about that.

<sup>also please make f1 work hitting a small button with a mouse is slow</sup>
